post/webnote-extension/index.md

```diff
@@ -0,0 +1,658 @@
+    },
+```
+
+![Login](./images/login.png)
+
+## Setup Home Page
+
+```html
+<template>
+    <div>
+        <div
+            style="
+                display: flex;
+                justify-content: space-between;
+                align-items: center;
+            "
+        >
+            <p
+                @click="logout"
+                style="
+                    text-decoration: underline;
+                    cursor: pointer;
+                    text-align: right;
+                "
+            >
+                Logout
+            </p>
+
+            <button
+                @click="$router.push({ name: 'upsert', params: { id: '+' } })"
+            >
+                ⊕
+            </button>
+        </div>
+        <p v-if="loading">Loading...</p>
+        <div
+            v-else
+            style="background: #222; padding: 16px; border-radius: 24px"
+        >
+            <li
+                v-for="note in notes"
+                :key="`note-${note.id}`"
+                style="
+                    display: flex;
+                    justify-content: space-between;
+                    align-items: center;
+                "
+            >
+                <div>
+                    <a :href="note.website"> {{ note.website }}</a>
+                    <div>{{ note.note }}</div>
+                </div>
+                <button
+                    @click="
+                        $router.push({
+                            name: 'upsert',
+                            params: { id: note.id },
+                        })
+                    "
+                >
+                    📝
+                </button>
+                <button @click="remove(note.id)">🗑️</button>
+            </li>
+        </div>
+    </div>
+</template>
+
+<script>
+    import { readItems, deleteItem } from "@directus/sdk";
+    import Cookies from "js-cookie";
+
+    export default {
+        inject: ["directus"],
+        data() {
+            return {
+                notes: null,
+                loading: false,
+            };
+        },
+        created() {
+            this.getNotes();
+        },
+        methods: {
+            async remove(id) {
+                await this.directus.request(deleteItem("notes", id));
+                this.getNotes();
+            },
+            logout() {
+                Cookies.remove("directus_auth");
+                this.$router.push({ name: "login" });
+            },
+            async getNotes() {
+                this.loading = true;
+                this.notes = await this.directus.request(readItems("notes"));
+                this.loading = false;
+            },
+        },
+    };
+</script>
+```
+
+### Load `home.vue` in `router.js`
+
+```js
+import HomeView from "../views/home.vue";
+```
+
+```js
+
+    {
+    path: "/",
+    name: "home",
+    meta: { public: false },
+     component: HomeView
+    },
+
+```
+
+![Home Page](./images/homepage.png)
+
+## Setup Create Note And Edit Note
+
```

Comment:
You need to add some detail here about how this works - it's one view for create and edit. What is the file name, etc?

Reply:
No. Its good. Let me know if there is any problem here.

Reply:
There is. You jump straight into code - it's needs some explanation.

Reply:
@phazonoverload ✅ Explanation added